# inputs
float64 distance
---
# outputs
bool result